R/Design-methods.R

```diff
@@ -1827,23 +1827,12 @@ setMethod("simulate",
                                  data=thisData,
                                  SIM=TRUE)
 
-                  thisDose <- NEXT$nextdose
-
-
-                  thisTDtargetDuringTrial<- NEXT$TDtargetDuringTrialEstimate
-
-
-                  thisTDtargetEndOfTrial<- NEXT$TDtargetEndOfTrialEstimate
-
-                  thisratioTDEOT <- NEXT$ratioTDEOT
-
-                  thisCITDEOT <- list(lower=NEXT$CITDEOT[1],
-                                      upper=NEXT$CITDEOT[2])
-
-
-                  thisTDtargetEndOfTrialatdoseGrid <- NEXT$TDtargetEndOfTrialAtDoseGrid
-
-
+                  thisDose <- NEXT$next_dose_drt
```

Comment:
when you clean up the `SIM` variable everywhere please also clean up the `NEXT` variable everywhere at the same time, thanks!